### ISSUE TITLE
[Question] The entropy value is a negative number, and the entropy loss is a positive number.

### ISSUE THREAD

**Issue by YSAA1** [DLR-RM/stable-baselines3#2004]
### ❓ Question

hello  I am confused about the use of PPO algorithm, and make some simple changes to PPO algorithm, such as adding dynamic entropy coefficient and so on. However, I have monitored from tensorboard that the entropy loss term of PPO has gradually changed from negative to positive. My understanding is that entropy cannot be negative, so negative entropy loss cannot be positive either. I checked the PPO source code for entropy.   

```python            
                if entropy is None:
                    entropy_loss = -th.mean(-log_prob)
                else:
                    entropy_loss = -th.mean(entropy)
                entropy_losses.append(entropy_loss.item())
```  
                
                

                
                

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by YSAA1** [comment]
![Uploading Snipaste_2024-09-13_15-03-41.png…]()

**Comment by YSAA1** [comment]
entropy loss fig

**Comment by YSAA1** [comment]
![Snipaste_2024-09-13_15-03-41](https://github.com/user-attachments/assets/f47335b5-acb7-4676-ab21-675e00347d66) 

![image](https://github.com/user-attachments/assets/ca619006-d2af-43e4-b68b-00ca4093ed52)

**Comment by araffin** [comment]
> My understanding is that entropy cannot be negative,

why not?
https://en.wikipedia.org/wiki/Differential_entropy

**Comment by YSAA1** [comment]
ok,thank u
